django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
Maybe it makes more sense then to add a 'empty_value_display' property to the Field and get that (with fallback to ModelAdmin and AdminSite).